inventory/hosts.example

```diff
@@ -838,8 +838,13 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # interface other than the default network interface.
 #openshift_set_node_ip=True
 
-# Configure dnsIP in the node config
-#openshift_dns_ip=172.30.0.1
+# Configure dnsIP in the node config.
+# This setting overrides the IP address used by dnsmasq, default value would be set to 
```

Comment:
"This setting overrides the bind IP address used by each node's dnsmasq.  By default, this value is set to the IP which ansible uses to connect to the node. Only update this variable if you need to bind dnsmasq on a different interface"